### ISSUE TITLE
Tk overwrites Base.Text

### ISSUE THREAD

**Issue by Varanas** [JuliaLang/Tk.jl#117]
which causes issues when using Atom.jl when importing Tk.jl in user code.

Ref: https://github.com/JunoLab/atom-julia-client/issues/118

The two solutions I see would be to either
- not export `Text` and require the user to use `Tk.Text` or maybe define `TkText` or
- make `Widget` not include `AbstractString` and instead use `TkWidget(x::AbstractString)` or something similar.

**Comment by Nectarineimp** [comment]
I vote for TkText definition. Currently including Tk breaks way too many other things by overriding Base.Text.

**Comment by tkelman** [comment]
Why is Base even exporting `Text` ? That seems internal to the Markdown parser and not a type users should need to work with very often.

**Comment by pfitzseb** [comment]
Right, so should we rename `Tk.Text`, unexport it from `Tk` or unexport `Text` from Base?

**Comment by tkelman** [comment]
All of the above?
